### PR TITLE
Trams now allowing for legal requirements on transfers

### PR DESCRIPTION
**Pull request by dneed-nimble**

### CODE REVIEW

File: TramsDataApi.Test/Factories/AcademyTransferProjectResponseFactoryTests.cs [hunk 1]
```diff
@@ -181,6 +190,13 @@ public void ReturnsAnAcademyTransferProjectResponse_WhenGivenACompleteAcademyTra
                     IsCompleted = academyTransferProjectModel.BenefitsSectionIsCompleted,
                     AnyRisks = academyTransferProjectModel.AnyRisks
                 },
+                LegalRequirements = new AcademyTransferProjectLegalRequirementsResponse
```
Comment (jordan-beard) [inline TramsDataApi.Test/Factories/AcademyTransferProjectResponseFactoryTests.cs:193]: This is probably what we should've done for conversions.

File: TramsDataApi/Migrations/TramsDb/20220627133721_NTI_UnderConsideration.Designer.cs [hunk 2]
```diff
@@ -838,7 +838,9 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                     b.Property<string>("Author")
                         .HasColumnType("nvarchar(max)");
 
-                    b.Property<bool?>("BenefitsSectionIsCompleted")
+                    b.Property<bool?>("" +
```
Comment (jordan-beard) [inline TramsDataApi/Migrations/TramsDb/20220627133721_NTI_UnderConsideration.Designer.cs:841]: This is a bit weird.

Comment (dneed-nimble) [reply TramsDataApi/Migrations/TramsDb/20220627133721_NTI_UnderConsideration.Designer.cs:841]: Strange it shows changes here, this is a migration from four months ago.

**Comment by jordan-beard** [comment]
Can you confirm that you've applied the migration to your local db and run a test in swagger to retrieve and update the legal requirements.

**Review by jordan-beard — APPROVED** [review]

**Pull request merged**